### PR TITLE
HZN-632 added setTrapProcessor and HZN-637 Initial check-in of files

### PR DESCRIPTION
**Pull request by pavanrp1**
Changes made for:

HZN-632: added setTrapProcessor in TrapNotificationHandlerDefaultImpl
HZN-637: Initial check-in of files (blueprint-trapd-handler-default.xml, applicationContext-trapDaemon.xml )

### CODE REVIEW

File: features/events/traps/src/main/resources/META-INF/opennms/applicationContext-trapDaemon.xml [hunk 1]
```diff
@@ -23,6 +23,19 @@
     <property name="eventReader" ref="eventReader"/>
   </bean>
   
+  <bean id="trapNotificationHandler" class="org.opennms.netmgt.trapd.TrapNotificationHandler"/>
+  
+  <bean id="trapReceiverSnmp4jImpl" class="org.opennms.netmgt.trapd.TrapReceiverSnmp4jImpl">
```
Comment (soleger) [inline features/events/traps/src/main/resources/META-INF/opennms/applicationContext-trapDaemon.xml]: There is already a TrapReceiverSnmp4jImpl bean in the file. You need to combine your new changes with it.

File: features/events/traps/src/main/resources/META-INF/opennms/applicationContext-trapDaemon.xml [hunk 2]
```diff
@@ -23,6 +23,19 @@
     <property name="eventReader" ref="eventReader"/>
   </bean>
   
+  <bean id="trapNotificationHandler" class="org.opennms.netmgt.trapd.TrapNotificationHandler"/>
```
Comment (soleger) [inline features/events/traps/src/main/resources/META-INF/opennms/applicationContext-trapDaemon.xml]: This is an interface. You cannot create a new one here. The handler that you should be using is the TrapNotificationHandlerDefaultImpl bean that is defined below.

File: features/events/traps/src/main/resources/META-INF/opennms/applicationContext-trapDaemon.xml (hunk 1 above)
Comment (pavanrp1) [inline features/events/traps/src/main/resources/META-INF/opennms/applicationContext-trapDaemon.xml]: oh yeah, somehow i missed it. Used existing bean.

File: features/events/traps/src/main/resources/META-INF/opennms/applicationContext-trapDaemon.xml (hunk 2 above)
Comment (pavanrp1) [inline features/events/traps/src/main/resources/META-INF/opennms/applicationContext-trapDaemon.xml]: Updated, using TrapNotificationHandlerDefaultImpl.

**Pull request merged**